site/src/pages/release-notes/1.3.0.mdx

```diff
@@ -227,6 +227,15 @@ date: 2020-11-30
 - `ServletRequest.getProtocol()` now returns the proper value when using <type://TomcatService> and
   <type://JettyService>. #3194
 - The route decorators are now evaluated in the reverse order they applied. #3160 #3166
+  - The purpose of this change is to make sure we provide a consistent experience when decorating a service.
+    For example, when we decorate a service like the following:
+    ```java
+    myService
+      .decorate(decoratorA)
+      .decorate(decoratorB);
+    ```
+    we expect `decoratorA` decorates `myService` and `decoratorB` decorates `decoratorA`.
```

Comment:
?
```suggestion
    We expect `decoratorA` decorates `myService` and `decoratorB` decorates `decoratorA`.
```

Reply:
It's actually a continuation of `the following` above.